src/aggregate.rs

```diff
@@ -64,6 +62,9 @@ use crate::DomainEvent;
 /// ```
 #[async_trait]
 pub trait Aggregate: Default + Serialize + DeserializeOwned + Sync + Send {
+    /// The aggregate type is used as the unique identifier for this aggregate and its events.
+    /// This is used for persisting the events and snapshots to a database.
+    const TYPE: &'static str;
```

Comment:
note that the 'static' can be removed in later versions of Rust since it's now the default lifetime for constants, however this would require bumping the MSRV so i'll leave it for now.

Reply:
This is a great idea but we need to add this in some examples.

Reply:
there are examples in the 'doc' module and in some doc comments (basically it's as well documented as the old approach was)